decoimpact/business/entities/rules/formula_rule.py

```diff
@@ -32,15 +32,8 @@ class FormulaRule(RuleBase, IMultiCellBasedRule):
 
     formula_output_name: str = "formula_result"
 
```

Comment:
I guess it is part of the guidelines, but I am not sure I find a compact hader like it is now an improvement.